docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+Hierbij gaan we uit van een, op basis van de visie op een gegevenslandschap, radicaal anders opgezet applicatielandschap. Daarin zijn veel uitdagingen die aanleiding hebben gegeven tot het zaakgericht werken veel integraler in de applicaties en registraties verwerkt. Voorbeelden van dergelijke uitdagingen zijn het verbeteren van de dienstverlening, een beter inzicht in proces- en voortgang uit back-office applicaties en digitaal uniform archiveren etc.. En zaakinformatie als gevolg niet meer op één centrale plats zit, maar gedistribueerd over meerdere registraties.
+
+Opmerking AK Is dat zo? Kern is en blijft toch de ZRC?! De kernregistratie zaken. Zaakinfo zit dan nog steeds op één plek. Aan zaken gerelateerde info, zoals gegevens van betrokkenen, van medewerkers en van objecten waarop zaken betrekking hebben, ja, die zitten elders.
+Grof geschetst bewegen we van een situatie zoals in figuur 1 naar een situatie zoals in figuur 2.
+
+### 1.3.2. Huidige Situatie
+
+In figuur 1 staat een schets van de huidige situatie zoals min of meer beschreven door GEMMA 2.
+
+![zgw-in-gemma2.png](./bestanden/zgw-in-gemma2.png?raw=true)
+*Figuur 1 - ZGW in de huidige situatie (~GEMMA 2)*
+
+* Een zaak is geregistreerd in een ZRC. Aan een zaak gerelateerde objecten (RGBZ) worden ook geregistreerd in de ZRC: betrokkenen, besluiten, klantcontacten, zaakobjecten en eventuele zaaktypespecifieke objecten met zaaktypespecifieke eigenschappen.
+* Vanuit een zaak liggen er verwijzingen naar zaaktype (ZTRC), Document vcq. informatieobject (DRC) en eventuele objecten (in theorie de Basisregistratie, maar vaak Gegevensmagazijn)
+* Voor betrokkene is in theorie ook sprake van een verwijzing (naar BRP en HR, kernregistratie medewerkers, kernregistratie bedrijven, kernregistratie klanten,..), maar in de praktijk wordt dit vaak in de ZRC zelf opgeslagen.
+* De Zaakafhandelcomponent bevat nog veel gegevens:
+    * procesdefinities (hoe moet een proces lopen, welke keuzes zijn er mogelijk);
+    * procesgegevens (waar in het proces zitten we, welke keuzes zijn in het proces gemaakt, wie is de behandelaar, welke gegevens heeft hij gebruikt voor welke processtap, ..)
+    * domein/objectgegevens: gegevens over objecten in het domein waartoe het proces behoort. Bijvoorbeeld voor een Subsidiesysteem de gegevens over een aangevraagde subsidie, doelen die men met subsidies wil stimuleren en de voorwaarden en uitnutting van diverse subsidiepotjes.
+
+* “Aanvraag” mist in dit plaatje. Dat klopt ook. Deze maakt onderdeel uit van de domein/objectgegevens. In de huidige praktijk wordt een aanvraag ook nog vaak (onterecht) met een zaak vereenzelvigd. Waarbij het in de vorm van een Informatieobject als PDFje wordt opgeslagen bij een zaak.
```

Comment:
Wat is nu precies de argumentatie om van een aanvraag niet direct een zaak te maken?
Is dit altijd zo, of alleen in bepaalde situaties omdat de aanvraag zelf niet gestructureerd genoeg is om te weten hoe de afhandeling zou moeten plaatsvinden.

Reply:
één melding/aanvraag kan tot meerdere zaken leiden. of andersom.

Bovendien wil je van een melding méér gegevens vastleggen dan je bij een zaak kwijt kunt. Allerlei meldingspecifieke gegevens die je wel gestructureerd in een formulier intypt, maar die je niet (of enkel als een document) bij een zaak kunt registreren..